src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -2446,7 +2446,7 @@ namespace HVACUnitarySystem {
 			PrintFlag = true;
 			TempSize = UnitarySystem( UnitarySysNum ).DesignMaxOutletTemp;
 			FieldNum = 18; // N18, \field Maximum Outdoor Dry-Bulb Temperature for Supplemental Heater Operation
-			SizingString = UnitarySystemNumericFields( UnitarySysNum ).FieldNames( FieldNum ) + " [m3/s]";
+			SizingString = UnitarySystemNumericFields( UnitarySysNum ).FieldNames( FieldNum ) + " [C]";
```

Comment:
Good catch, but it could be that it needs to be 
  
	TempSize = UnitarySystem( UnitarySysNum ).MaxOATSuppHeat;